h/static/scripts/test/time-test.js

```diff
@@ -50,6 +51,34 @@ describe('time', function () {
   });
 
   describe('.toFuzzyString', function () {
+
+    var originalIntl;
+
+    before(function () {
+
+      // Intl isn't implemented in PhantomJS so we have to mock it in these
+      // tests. The mock version returns the results we'd expect from the real
+      // one for the test values that we use.
+      originalIntl = window.Intl;
```

Comment:
In this case, the chain of functions isn't very long so either (1) or (3) would work. You could implement (3) by making `format()` a separately unit-testable function and an argument to `toFuzzyString()`